### PR TITLE
remove outdated local help page, link to docs instead

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**
Title

### CODE REVIEW

**Review by DL6ER — APPROVED** [review]

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -668,14 +668,12 @@ function pidofFTL()
...
                 </li>
-                <?php if($auth){ ?>
-                <!-- Help -->
-                <li<?php if($scriptname === "help.php"){ ?> class="active"<?php } ?>>
-                    <a href="help.php">
-                        <i class="fa fa-question-circle"></i> <span>Help</span>
+                 <!-- Docs -->
+                 <li>
+                    <a href="https://docs.pi-hole.net/" rel="noopener" target="_blank">
+                        <i class="fa-question-circle"></i> <span>Donate</span>
```
Comment (XhmikosR) [inline scripts/pi-hole/php/header.php:674]: This should have been `fa fa-question-circle` and not say `Donate` :P

**Comment by yubiuser** [comment]
If this is not an interim solution I'd like to suggest two things:
1. Remove help.php also
2. Copy the content of the help page to the online doc. At the moment there is no explanation about the Web Interface and inexperienced users might need that.

**Comment by XhmikosR** [comment]
help.php is removed already.

**Comment by XhmikosR** [comment]
I would also argue about the page removal. It wasn't that bad...

**Pull request merged**